content/en/infrastructure/resource_catalog/resource_changes/_index.md

```diff
@@ -0,0 +1,250 @@
+  text: "Google Cloud Platform Integration for Resource Changes"
+- link: "/integrations/azure/"
+  tag: "Documentation"
+  text: "Azure Integration for Resource Changes"
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">Resource Changes is not supported for your selected <a href="/getting_started/site">Datadog site</a> ({{< region-param key="dd_site_name" >}}).</div>
+{{< /site-region >}}
+
+{{< callout url="https://www.datadoghq.com/product-preview/recent-changes-tab/" >}}
+  esource Changes is in Preview. Click <strong>Request Access</strong> and complete the form to request access.
+{{< /callout >}} 
+
+## Overview
+
+Resource Changes provides visibility and control over configuration changes to your cloud infrastructure. It helps you monitor modifications to resources, aiding in troubleshooting incidents and understanding the evolution of your environment.
+
+{{< img src="/infrastructure/resource_catalog/resource-changes.png" alt="Datadog Resource Changes interface showing a list of infrastructure configuration changes. The screen displays a VM instance named \"vm-new-jmcintyre-kafka\" with a StorageProfile update, including a side-by-side diff view highlighting changes in JSON format. The table shows multiple resources with timestamps, change types (mostly \"UPDATE\"), and details of the modifications. Filters are available at the top for cloud, region, environment, and other attributes." width="100%">}}
+
+Resource Changes supports a variety of resources, including:
+- Hosts
+- Containers
+- Serverless Functions
+- Databases
+- Queues
+- Storage
+- Networking
+- Monitoring
+- Identity
+- Data Processing
+
+For a comprehensive list of supported resources, see the [Supported resources](#supported-resources) section.
+
+## Setup
+
+To view configuration changes for your resources:
+
+1.  **Enable Resource Collection**: Ensure that resource collection is enabled for your cloud resources so they are visible within the Resource Catalog. You can manage this setting from the [Resource Catalog settings page][2] or the relevant [cloud provider integration tile][3]. This step automatically enables [Snapshot Changes](#snapshot-changes) for your resources.
+2.  **Cloud Provider Event Forwarding**: For real-time or more frequent change updates, configure event forwarding from your cloud provider:
+    - **AWS**: Set up change event forwarding through AWS Config. See the [AWS Config integration page][4] for details.
+    - **Azure**: Enable Resource Collection for your Azure subscriptions in the [Azure integration tile][5].
+    - **Google Cloud Platform**: Follow the instructions on the [Google Cloud Platform integration page][6] to receive resource events in Datadog when Google's Cloud Asset Inventory detects changes in your cloud resources.
+
+ <div class="alert alert-info">If you only enable Resource Collection without specific cloud provider event forwarding, Datadog uses <strong>Snapshot Changes</strong>.</div>
```

Comment:
```suggestion
 <div class="alert alert-info">If you enable only Resource Collection without setting up cloud provider event forwarding, Datadog identifies configuration changes using <strong>Snapshot Changes</strong>.</div>
```

Reply:
My suggestion to make this note a little clearer. Might not be needed based on the other comment.